### ISSUE TITLE
dataset files missing

### ISSUE THREAD

**Issue by Highpressure** [padmalcom/Real-Time-Voice-Cloning-German#6]
I am not able to find your dataset files

**when running** 
python synthesizer_train.py my_run E:\Datasets\

**it asks me for** 
FileNotFoundError: [Errno 2] No such file or directory: 'E:\\Datasets\\train.txt'

**when running**
python synthesizer_preprocess_embeds.py E:\Datasets\SV2TTS\synthesizer

**it asks me for**
Arguments:
    synthesizer_root:      E:\Datasets\SV2TTS\synthesizer
    encoder_model_fpath:   encoder\saved_models\pretrained.pt
    n_processes:           4

Traceback (most recent call last):
  File "synthesizer_preprocess_embeds.py", line 25, in <module>
    create_embeddings(**vars(args))
  File "E:\DeepFakes\Real-Time-Voice-Cloning\synthesizer\preprocess.py", line 260, in create_embeddings
    assert wav_dir.exists() and metadata_fpath.exists()
AssertionError

**when running** 
python encoder_preprocess.py E:\Datasets

**it looks for - and does nothing**
Arguments:
    datasets_root:   E:\Datasets
    out_dir:         E:\Datasets\SV2TTS\encoder
    datasets:        ['librispeech_other', 'voxceleb1', 'voxceleb2', 'm-ailabs']
    skip_existing:   False

Preprocessing librispeech_other
Couldn't find E:\Datasets\LibriSpeech\train-other-500, skipping this dataset.
Preprocessing voxceleb1
Couldn't find E:\Datasets\VoxCeleb1, skipping this dataset.
Preprocessing voxceleb2
Couldn't find E:\Datasets\VoxCeleb2, skipping this dataset.
Preprocessing m-ailabs
Couldn't find E:\Datasets\de_DE\by_book, skipping this dataset.

**Comment by padmalcom** [comment]
Try to understand what you are doing, you are loading a file that does not exist. Pass the path where you keep your training data which is obviously not E:\Datasets\

**Comment by Highpressure** [comment]
i have an e: drive too and set everything up from the pathes as you.
I did use https://github.com/padmalcom/Real-Time-Voice-Cloning-German/releases/tag/0.1 and put the files in the corresponding directories

I am new to real-time-voice-cloning and tried to start with german right away :)

**Comment by padmalcom** [comment]
So you want to use the German mailabs training data. Look at the last line of the error message you get. So you see what directory structure the encoder expects?

**Comment by Highpressure** [comment]
ok... I first thought I dont need the german audiofiles, so that the pretrained set is all I need?
I now started from scratch and downloaded the full de_DE by book files and powershelled the .txt structure in there now myself.
got myself also the train 100 and train 500 datasets.
my goal was it to create my own "virtual voice" of myself. from what I have read so far I should have lots of hours of book readings.
(so yes, this is kinda my first step into the whole world of ai/ml/deep learning)

still i think its complex and not very well documented what when how to use (not your branch, in general I mean).
feeling little lost and the first results are ... meh, not compareable to some stuff I saw on youtube.

thanks for the help anyhow :)
